python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -284,6 +315,99 @@ def _process_adapter_response_metadata(
 
         return processed_adapter_response
 
+    def _process_lineage_metadata(
+        self,
+        manifest: Mapping[str, Any],
+        dagster_dbt_translator: DagsterDbtTranslator,
+        target_path: Path,
+    ) -> Dict[str, Any]:
```

Comment:
Nitpick: preferable to return `Mapping` over `Dict`

Reply:
Having `Dict` as a return type is fine, since we are exactly returning that type. But arguments should definitely be typed as `Mapping`: https://docs.python.org/3/library/typing.html#typing.Dict

Reply:
I don't feel super strongly in this instance, so feel free to override.